### DISCUSSION TITLE
26.4. Käyttöhäiriö

### DISCUSSION THREAD

**Discussion by emta001** [KohaSuomi/Koha#520] [Huolto- ja häiriötiedotteet]
Yhteys palvelimeen katkesi noin klo 15.00 sekä Vaara- että Kyyti-kirjastoissa. Molemmat kimpat sijaitsevat samalla palvelimella, jonka levyjärjestelmä on nyt jumissa. Asiaa selvitellään vielä ja selvittelyn etenemisestä tiedotetaan tähän keskusteluun.

**Comment by CodoDerDritte** [comment]
Ongelma vaikuttaa ikävä kyllä leviävän. Mikään edustapalvelin ei nyt vastaa, joten kaikki Kohat ovat pois pelistä.

**Comment by CodoDerDritte** [comment]
Toipuminen on alkanut. Node 2 eli Jo otti levyjärjestelmän panttivangiksi ja se aiheutti häiriöitä, koska mikään muu edustanode ei päässyt kirjoittamaan sinne. Kohat palautuvat pikkuhiljaa, mutta muille palvelimille kertyneiden i/o -jonojen purkaminen voi viedä hetken aikaa, joten hitautta saattaa ilmetä. Vaara, Kyyti ja Kirkes-testi edelleen nurin ja Jo:n ongelmanselvitys jatkuu.

**Comment by CodoDerDritte** [comment]
Vaara on toipunut.

**Comment by CodoDerDritte** [comment]
ja Kyyti myös.

**Comment by CodoDerDritte** [comment]
Ongelman syy oli Jo-palvelimen känkkäränkkäkohtaus, jonka yhteydessä se vei kaikilta muilta palvelimita lelut ja sulkeutui niiden kanssa omaan leikkimökkiinsä. Olemme komentaneet Jo:n nurkkaan häpeämään ja miettimään käytöstään.

(Vakavammin puhuen meillä ei ole mitään käryä mikä on ongelman juurisyy eikä selvittäminenkään oikein onnistu, koska palvelin jouduttiin boottaamaan).

**Comment by CodoDerDritte** [comment]
Kaikki palvelut ovat nyt palautuneet normaaleiksi, mukaanlukien testit ja nextit. Vaaralle ja Kyytille ajastan ensi yöksi indeksoinnin, koska indeksiin saattoi tässä kolarissa tulla hieman lommoja.

Häiriön kesto oli Vaarassa ja Kyytissä noin 1.5 tuntia ja Outissa, Vaskissa, Hellessä ja Lapissa noin puoli tuntia. Ilmeisesti Siilinjärvellä ja Lumpeissa häiriö ei ehtinyt näkyä ollenkaan. Pahoittelemme häiriötä!
